tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py

```diff
@@ -0,0 +1,340 @@
+        # Forward Series Flags = 0x04:
+        # - Bit 2 = 1, MAC Data Request
+        # - Bits 0,1 & 3-7 = 0
+        # Concatenation of Link Metric Type ID Flags = 0x00:
+        # - Item1: (0)(1)(000)(000) = 0x40
+        # -- E = 0
+        # -- L = 1
+        # -- Type/Average Enum = 0 (count)
+        # -- Metrics Enum = 0 (count)
+        self.nodes[SED_1].link_metrics_mgmt_req_forward_tracking_series(leader_addr, SERIES_ID, 'r', 'p')
+
+        self.simulator.go(5)
+
+        # Step 6 - Configures SED_1 to send MAC Data Request every 2 seconds and wait for 30 seconds
+        self.simulator.go(30)
+
+        # Step 7 - SED_1 sends an MLE Data Request to retrieve aggregated Forward Series Results
+        self.nodes[SED_1].link_metrics_query_forward_tracking_series(leader_addr, SERIES_ID)
+
+        self.simulator.go(5)
+
+        # Step 9 - SED_1 clears the Forward Tracking Series
+        # Forward Series Flags = 0x00:
+        # - Bits 0-7 = 0
+        # Concatenation of Link Metric Type ID Flags = NULL:
+        self.nodes[SED_1].link_metrics_mgmt_req_forward_tracking_series(leader_addr, SERIES_ID, 'X', '')
+
+        self.simulator.go(5)
+
+        # Step 16 - SSED_1 requests a Forward Tracking Series by sending a Link Metrics Management Request
+        # Forward Series Flags = 0x02:
+        # - Bit 1 = 1, (all Link Layer Data Frames)
+        # - Bits 0, 2-7 = 0
+        # Concatenation of Link Metric Type ID Flags = 0x0a:
+        # - Item1: (0)(0)(001)(010) = 0x0a
+        # -- E = 0
+        # -- L = 0
+        # -- Type/Average Enum = 1 (Exponential Moving Average)
+        # -- Metrics Enum = 2 (Link Margin)
+        self.nodes[SSED_1].link_metrics_mgmt_req_forward_tracking_series(leader_addr, SERIES_ID_2, 'd', 'm')
+
+        self.simulator.go(5)
+
+        # Step 18 - SSED_1 sends MAC Data packet every 1 second for 30 seconds
+        # Note: As the api to send MAC Data frame hasn't been implemented, we send MLE Link Probe message here
+        for i in range(30):
+            self.nodes[SSED_1].link_metrics_send_link_probe(leader_addr, SERIES_ID_2, 1)
+            self.simulator.go(1)
+
+        # Step 19 - SSED_1 sends an MLE Data Request to retrieve aggregated Forward Series Results
+        self.nodes[SSED_1].link_metrics_query_forward_tracking_series(leader_addr, SERIES_ID_2)
+
+        self.simulator.go(5)
+
+        # Step 21 - SSED_1 clears the Forward Series Link Metrics
+        # Forward Series Flags = 0x00:
+        # - Bits 0-7 = 0
+        # Concatenation of Link Metric Type ID Flags = NULL
+        self.nodes[SSED_1].link_metrics_mgmt_req_forward_tracking_series(leader_addr, SERIES_ID_2, 'X', '')
+
+        self.simulator.go(5)
+
+        # Step 23 - SSED_1 sends an MLE Data Request to retrieve aggregated Forward Series Results
+        self.nodes[SSED_1].link_metrics_query_forward_tracking_series(leader_addr, SERIES_ID_2)
```

Comment:
If following Test Plan exactly (`Query ID = [same value as set in step 9]`), here should use `SERIES_ID`.
Or maybe the Series/Query ID in step 19/21/23 in TP be updated to the same one as Step 16?